tfhe/src/integer/gpu/server_key/radix/tests.rs

```diff
@@ -1825,6 +1923,100 @@ where
     }
 }
 
+fn integer_scalar_eq<P>(param: P)
+where
+    P: Into<PBSParameters> + Copy,
+{
+    let gpu_index = 0;
+    let device = CudaDevice::new(gpu_index);
+    let stream = CudaStream::new_unchecked(device);
+
+    let (_, h_sks) = KEY_CACHE.get_from_params(param, IntegerKeyKind::Radix);
```

Comment:
I think h_sks is not used and you can remove the associated import to KEY_CACHE no?